apps/docs/content/guides/auth/debugging/error-codes.mdx

```diff
@@ -11,7 +20,10 @@ Errors originating from the server API classed as `AuthApiError` always have a `
 
 <AuthErrorCodesTable />
```

Comment:
We can probably move the status codes out similar to what storage does. Are we okay with classifying them as "Legacy Error Codes"?

Reply:
Are they actually legacy? Or are they always going to convey the same information (just that this information is too coarse-grained to be useful)? We could call them "HTTP status codes" and put them beneath the other error codes

Reply:
I think they convey the same information but the information is too coarse-grained. HTTP status codes sounds fine yup